analyzer/__load__.zeek

```diff
@@ -1 +1,2 @@
+@load-sigs ./dpd.sig
```

Comment:
We need to add `dpd.sig` to the `SCRIPTS` in analyzer/CMakeLists.txt`.

Reply:
Added